resources/web/signaldata/StandardCreator.js

```diff
@@ -0,0 +1,953 @@
+                LABKEY.SignalData.StandardCreator.sourcesStore = Ext4.create('LABKEY.ext4.data.Store', {
+//                   model: 'LABKEY.SignalData.StandardSource',
+                    schemaName: 'lists',
+                    queryName: 'SignalDataStandardSource'
+                });
+            }
+
+            if (doFilter) {
+                LABKEY.SignalData.StandardCreator.sourcesStore.filterArray = filters;
+                LABKEY.SignalData.StandardCreator.sourcesStore.load();
+            }
+            return LABKEY.SignalData.StandardCreator.sourcesStore;
+        }
+    },
+
+    initComponent : function() {
+
+        SC = this;
+        this.loadContext(this.context);
+
+        this.items = [
+            this.getInputs(), // west
+            this.getDefinitionForm(), // east
+            this.getStandardsDisplay() // center
+        ];
+
+        this.callParent();
+
+        this.on('selectsource', this.onSelectSource, this);
+        this.on('standardsave', this.onStandardSave, this);
+
+        this.pTask = new Ext4.util.DelayedTask(function() {
+            if (Ext4.isArray(this.definitions)) {
+                this.renderCalibrationStandards(this.definitions);
+            }
+        }, this);
+    },
+
+    plotTask : function() {
+        this.pTask.delay(500);
+    },
+
+    loadContext : function(context) {
+        this.rawInputs = context.rawInputs;
+    },
+
+    getInputs : function() {
+
+        return {
+            xtype: 'panel',
+            itemId: 'west',
+            region: 'west',
+            width: 250,
+            layout: {
+                type: 'vbox',
+                align: 'stretch'
+            },
+            items: [{
+                title: 'Choose a Standard to Define',
+                id: 'inputsgrid',
+                xtype: 'grid',
+                height: 400,
+                store: {
+                    xtype: 'store',
+                    model: 'LABKEY.SignalData.StandardSource',
+                    data: this.rawInputs
+                },
+                selModel: {
+                    selType: 'checkboxmodel',
+                    mode: 'MULTI'
+                },
+                columns: [{text: 'Inputs', dataIndex: 'name', flex: 1}],
+                hideHeaders: true,
+
+                // listeners
+                listeners: {
+                    viewready : function(g) {
+                        g.getStore().filter([{
+                            filterFn: function(item) {
+                                return item.get('name').indexOf('PRE_') == -1 && item.get('name').indexOf('POST_') == -1;
+                            }
+                        },{
+                            filterFn: function(item) {
+                                return item.get('name').indexOf('BLANK') == -1;
+                            }
+                        },{
+                            filterFn: function(item) {
```

Comment:
These filter functions looking for formulation prefixes should not be in signalData.